### DISCUSSION TITLE
Satellite Tracking Plug-In Questions

### DISCUSSION THREAD

**Discussion by irobot2000** [Stellarium/stellarium#3386] [General]
I noticed in version 23.1, I cannot make the orbit lines of artificial satellites appear. Earlier versions I've used worked fine.
Is there a setting I'm missing?

Also, there does not appear to be a way to update TLE's from a local file from within the interface. Is there a way to
do this manually? 

Thanks,
Mike

**Comment by alex-w** [comment]
> I noticed in version 23.1, I cannot make the orbit lines of artificial satellites appear. Earlier versions I've used worked fine.
Is there a setting I'm missing?

Please check settings in Satellites plugin (second tab - Satellites)

> Also, there does not appear to be a way to update TLE's from a local file from within the interface. Is there a way to
do this manually?

Please use standard protocol `file://` in path of location for TLE
